App.js

```diff
@@ -8,60 +8,66 @@
 
 import React from 'react';
 import {
-  SafeAreaView,
   StyleSheet,
-  ScrollView,
   View,
-  Text,
-  StatusBar,
   PermissionsAndroid,
   ImageBackground,
 } from 'react-native';
-import Geolocation from 'react-native-geolocation-service';
-import Weather from './components/Weather.js';
+import bgImage from './assets/background-images/sky.jpg';
+import Weather from './components/Weather';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  background: {
+    flex: 1,
+    flexDirection: 'column',
+  },
+  baseText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 18,
+  },
+});
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      hasLocationPermission: false,
```

Comment:
I more often meets this convention.
```suggestion
      isLocationPermission: false,
```

Reply:
The variable name has been changed